lib/models/ReplicationConfiguration.js

```diff
@@ -148,10 +148,10 @@ class ReplicationConfiguration {
         }
         const role = parsedRole[0];
         const rolesArr = role.split(',');
-        if (rolesArr.length !== 2) {
+        if (rolesArr.length > 2) {
```

Comment:
While you're at this, I'm thinking we should allow no role at all for zenko, since it will use a special account to do the operations, it does not make sense to set a dummy role for it. It may break AWS strict specs, and we may want to allow only if we're not using vault. @rahulreddy any concern?

Reply:
I agree that is preferable to setting a dummy role. However I think the scope of changes for replicating to AWS is already wide. Maybe we should add a TODO for modifying this along with changes related to zenko configuration?

Reply:
Yes you can add a TODO if you wish it can help tracking.